client/homescreen/test/index.js

```diff
@@ -101,26 +100,10 @@ describe( 'Homescreen Layout', () => {
 		expect( taskList ).toBeNull();
 	} );
 
-	it( 'should not show TaskList when it is complete', () => {
```

Comment:
Removed this test as it is not relevant anymore (taskListComplete not being passed in anymore).